src/clj/game/cards/upgrades.clj

```diff
@@ -369,6 +370,31 @@
                                                    (register-events state side (:events (card-def c)) c)))}
                                    card nil))}]}
 
+   "Giordano Memorial Field"
+   {:events
+    {:successful-run
+     {:interactive (req true)
+      :async true
+      :req (req this-server)
+      :effect (req (let [credits (:credit runner)
+                         cost (* 2 (count (:scored runner)))
+                         pay-str (str "Pay " cost " [Credits]")]
+                     (show-wait-prompt state :corp (str "Runner to pay " cost " [Credits]"))
+                     (continue-ability
+                       state :runner
+                       {:player :runner
+                        :async true
+                        :prompt (str pay-str " or end the run?")
```

Comment:
This should probably indicate that Memorial Field is the source of this choice.

Reply:
Good call.